plenoptic/synthesize/eigendistortion.py

```diff
@@ -351,11 +351,11 @@ def _synthesize_power(self,
             Fv = fisher_info_matrix_vector_product(y, x, v, _dummy_vec)
             Fv = Fv - shift * v  # minor component
 
-            v_new = torch.qr(Fv)[0] if k > 1 else Fv
+            v_new = torch.qr(Fv)[0]  # (ortho)normalize vector(s)
```

Comment:
When k=1 before, vector wasn't being normalized after each iter. 
Using QR, it's orthonormalized the same was as when k>1. But since there's only 1 vec, it's just being normalized bc there's no other vecs to orthogonalize against. There's no noticeable performance hit in using QR when k=1; and it's cleaner to just use the same method.